packages/@o3r/telemetry/src/cli/index.ts

```diff
@@ -0,0 +1,94 @@
+import { existsSync, readFileSync } from 'node:fs';
+import path from 'node:path';
+import type { Opts as MinimistOptions } from 'minimist';
+import { getEnvironmentInfo } from '../environment';
+import { type CliMetricData, sendData as defaultSendData, type SendDataFn } from '../sender';
+
+/** Simple Logger interface */
+interface Logger {
```

Comment:
Can get the interface from `@o3r/core`

Reply:
As discussed, we can't as core depends on telemetry